### PR TITLE
testManager added

### PR DESCRIPTION
**Pull request by GiorgioJuan**

### CODE REVIEW

File: dev/docker-compose.dev.yml [hunk 1]
```diff
@@ -55,6 +55,25 @@ services:
     volumes:
       - ruleManagerPostgres:/var/lib/postgresql/data
 
+  testManagerPostgres:
+    image: postgres:13
+    ports:
+      - "5437:5437"
+    env_file:
+      - .env
```
Comment (GonzaMato) [inline dev/docker-compose.dev.yml]: change this

**Review by GonzaMato — CHANGES_REQUESTED** [review]

**Review by GonzaMato — CHANGES_REQUESTED** [review]
testearlo

**Comment by Gonza69420** [comment]
hacela a dev

**Review by GonzaMato — APPROVED** [review]
Alt text

**Pull request merged**